src/services/redisSessionStorage.ts

```diff
@@ -48,19 +52,64 @@ export default class RedisSessionStorage implements ISessionStorage {
       );
     });
 
-    const [setSessionTokenResult, setWalletTokenResult] = await Promise.all([
+    const newSessionInfo: SessionInfo = {
+      sessionToken: user.session_token,
+      timestamp: new Date()
+    };
+    const setUserTokenInfo = new Promise<Either<Error, boolean>>(resolve => {
+      this.redisClient.set(
+        `${userKeyPrefix}${user.fiscal_code}-${sessionKeyPrefix}${
```

Comment:
I think we only need `${sessionInfoKeyPrefix}${user.session_token}` ?

Reply:
also you should compute this key only once and store it into a variable, to make sure that this one and the key stored in the set below are the same

Reply:
fixed in eeeac6d